ADA/databricks_fundamentals.qmd

```diff
@@ -66,107 +73,118 @@ The storage and computation are separated into different components rather than
 
 ------------------------------------------------------------------------
 
-There are a few different ways of storing files and data on DataBricks. Your data, and modelling areas will reside in the 'unity catalog', whereas your scripts and code will live on your 'workspace'.
+There are a few different ways of storing files and data on Databricks. Your data, and modelling areas will reside in the 'unity catalog', whereas your scripts and code will live on your 'workspace'.
 
 ### Unity catalog
 
-The majority of data and files on DataBricks should be stored in the 'unity catalog'. This is similar in concept to a traditional database server, however the unity catalog also contains file storage in the form of volumes.
+The majority of data and files on Databricks should be stored in the 'unity catalog'. This is similar in concept to a traditional database server, however the unity catalog also contains file storage in the form of volumes.
 
-The unity catalog can be accessed through the 'Catalog' option in the DataBricks sidebar.
+The unity catalog can be accessed through the 'Catalog' option in the Databricks sidebar.
 
 ![](/images/ada-unity-catalog-sidebar.png)
 
+#### Structure of the unity catalog
+
+There is one 'unity catalog' for the whole of the Department for Education, this is what enables the Department to keep track of all of it's data in a single place. The 'unity catalog' can contain any number of catalogs (equivalent to databases).
+
+A catalog can contain any number of schemas.
+
+A schema can contain any number of tables, views and volumes.
+
+![](/images/ada-unity-catalog.jpg)
+
 #### Catalogs not databases
 
-The unity catalog contains numerous catalogs, which are similar in concept to a SQL database.
+The 'unity catalog' is a single catalog that contains all the other catalogs of data in the Department. Catalogs are very similar in concept to a SQL database in that they they contain schemas, tables of data and views of data.
 
 #### Schemas, tables and views
 
 Like a SQL database a catalog has schemas, tables, and views which store data in a structured (usually tabular) format.
 
-This is where you would store your core datasets and pick up data to analyse from.
-
-#### Volumes
+A schema is a sub-division of a catalog which allows for logical separation of data stored in the catalog. Whoever creates a schema is it's owner, and is able to set fine grained permissions on who can see / edit the data within it. Permissions can also be set for groups of analysts, and can be modified by the ADA team if the original owner is no longer available.
```

Comment:
Is there any general guidance given by the ADA team, or have there been discussions about how a schema should be organised/maintained by teams? Are there examples of what good practice looks like?